wcfsetup/install/files/lib/system/image/adapter/ImagickImageAdapter.class.php

```diff
@@ -356,6 +356,16 @@ public function writeImage($image, $filename) {
 			fclose($file);
 		}
 		else {
```

Comment:
Should the `if` part of the condition be handled as well?

Reply:
We could simply remove that `if`, this is a 6 year old workaround for ancient PHP versions using ImageMagick 3.1.0RC1 from early 2012.

Reply:
I'd be fine with that.